extensions/notebook/src/book/bookTocManager.ts

```diff
@@ -150,7 +150,7 @@ export class BookTocManager implements IBookTocManager {
 		}
 
 		for (const [key, value] of this.tocFiles.entries()) {
-			const yamlFile = await yaml.safeLoad(value);
```

Comment:
this wasn't related to the package version update, but it was a red squiggly when I was in the file, so updated this while I was there